ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -228,6 +234,12 @@ func (h *FaultHandler) startNetworkBlackholePort(ctx context.Context, protocol,
 			})
 			return string(cmdOutput), err
 		}
+		logger.Info("[INFO] Successfully appended new rule to iptable chain", logger.Fields{
+			"command": appendRuleCmdString,
+			"output":  string(cmdOutput),
+			"taskArn": taskArn,
+			"error":   err,
```

Comment:
nit (non-blocking): same comment

Reply:
Addressed in [1e5cb61](https://github.com/aws/amazon-ecs-agent/commit/1e5cb61d395aeee30b0583c7cb0b30e339c6c4ef)